internal/reports/create.go

```diff
@@ -0,0 +1,50 @@
+package reports
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+)
+
+type CreateHandOff struct {
+	AwsAccountId      string
+	AwsHostedZoneName string
+	AwsRegion         string
+	ClusterName       string
+	ArgoCDUrl         string
+	ArgoCDUsername    string
+	ArgoCDPassword    string
+	VaultUrl          string
+	VaultToken        string
+}
+
+func BuildCreateHandOffReport(clusterData CreateHandOff) bytes.Buffer {
+
+	var handOffData bytes.Buffer
+	handOffData.WriteString(strings.Repeat("-", 70))
+	handOffData.WriteString(fmt.Sprintf("\nCluster %q is up and running!:\n", clusterData.ClusterName))
+	handOffData.WriteString(strings.Repeat("-", 70))
```

Comment:
Should we add a warn to press a button to leave the screen?

Reply:
IMHO the user level that we are aiming know that ctrl+c will quit